File: frontend/src/app/views/profile/race.ts
export class Race {

  public ID: number;
  public name: string;
  public place: string;
  public date: string;
  public prize: number;
  public horses: any [];
  public ISOVER: boolean;

  constructor(ID: number, name: string, place: string, date: string, prize: number, horses: any[], ISOVER: boolean) {
    this.ID = ID;
    this.name = name;
    this.place = place;
    this.date = date;
    this.prize = prize;
    this.horses = horses;
    this.ISOVER = ISOVER;
  }
}

File: servlet/src/main/java/unicyb/horseracingservice/entity/Race.java
package unicyb.horseracingservice.entity;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;
import java.util.Vector;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonPropertyOrder({"ID", "name", "place", "date", "prize", "horses", "ISOVER"})
public class Race {
    private int ID;
    private String name;
    private String place;
    private Timestamp date;
    private float prize;
    private Vector<Horse> horses;
    private Boolean ISOVER;
}
